foundations/javascript_basics/problem_solving.md

```diff
@@ -65,10 +65,10 @@ Many beginners try to solve the big problem in one go. **Don’t do this**. <spa
 
 In short, break the big problem down and solve each of the smaller problems until you've solved the big problem.
 
-### Solving Fizz Buzz
+### Solving fizz buzz
 To demonstrate this workflow in action, let's solve a common programming exercise: [FizzBuzz](https://en.wikipedia.org/wiki/Fizz_buzz).
```

Comment:
```suggestion
To demonstrate this workflow in action, let's solve [a common programming exercise: Fizz Buzz, explained in this wiki article](https://en.wikipedia.org/wiki/Fizz_buzz).
```

Updated for consistency with the header and to create a more descriptive link which is unrelated to this PR